app/views/helpers/pagination.php

```diff
@@ -37,7 +37,33 @@
 class PaginationHelper extends AppHelper
 {
     public $helpers = array('Paginator');
+    
+    /**
+     * Wraps Paginator->sort with default order
+     * 
+     * @param string    Title for the link
+     * @param string    The name of the key that the recordset should be sorted. If $key is null $title will be used for the key, and a title will be generated by inflection
+     * @param array     options; 'defaultOrders' => array('field' => 'defaultOrder') where defaultOrder = desc/asc
+     *
+     * @return string   the link to sort
+     */
+    public function sortDefaultOrder($title, $key = null, $options =  array()){
+        // direction is given or no default is specified
+        if(isset($this->params['direction']) 
+            || !isset($options['defaultOrders'])
+            || !isset($options['defaultOders'][$key])){
```

Comment:
There is a small typo here. 

`defaultOders` --> `defaultOrders` (`r` missing in `Orders`)